### PR TITLE
feat: yet another syntax for actor class management, @ggreif version

### PR DESCRIPTION
**Pull request by crusso**
```
'(' 'system' <exp_post> '.' <id> ')'`
```

E.g.

```
await (system Lib.Node)(#upgrade n)(i); // upgrade!
```

where `system <exp>.<id>` abbreviates `<exp>.system.id`.

### CODE REVIEW

**Review by ggreif — APPROVED** [review]
Great!

File: doc/md/language-manual.md [hunk 1]
```diff
@@ -1067,7 +1067,76 @@ module {
 }
 ```
 
-On the Internet Computer, if this library is imported as identifier `Lib`, then calling `await Lib.<id>(<exp1>, …​, <expn>)`, installs a fresh instance of the actor class as an isolated IC canister, passing the values of `<exp1>`, …​, `<expn>` as installation arguments, and returns a reference to a (remote) actor of *type* `Lib.<id>`, that is, `T`. Installation is (necessarily) asynchronous.
+On the Internet Computer, if this library is imported as identifier `Lib`, then calling `await Lib.<id>(<exp1>, ..., <expn>)`, installs a fresh instance of the actor class as an isolated IC canister, passing the values of `<exp1>`, ...​, `<expn>` as installation arguments, and returns a reference to a (remote) actor of *type* `Lib.<id>`, that is, `T`. Installation is (necessarily) asynchronous.
```
Comment (ggreif) [inline doc/md/language-manual.md:1070]: elsewhere you keep using the ellipsis unicode character

File: doc/md/actor-classes.md [hunk 2]
```diff
@@ -55,3 +55,23 @@ actor class Map(n : Nat) {
...
+
+## Configuring and managing actor class instances
+
+On the Internet Computer, the primary constructor of an imported actor class always creates a new principal and installs a fresh instance of the class as the code for that principal.
+
+To provide further control over actor class installation, Motoko endows each imported actor class with an extra, secondary constructor.
+This constructor takes an additional first argument that specifies the desired installation mode. The constructor is only available via special syntax that stresses its
+`system` functionality.
+
+Using this syntax, its possible to specify initial canister settings (such as an array of controllers), manually install, upgrade and reinstall canisters, exposing all of the
```
Comment (ggreif) [inline doc/md/actor-classes.md:73]:
```suggestion
Using this syntax, it's possible to specify initial canister settings (such as an array of controllers), manually install, upgrade and reinstall canisters, exposing all of the
```

File: doc/md/language-manual.md [hunk 3]
```diff
@@ -1055,7 +1055,7 @@ module {
 
 provided that:
 
--   the actor class declaration `<dec>` has function type `(U1, …​, Un) → async T` under the static environment induced by the imports in `<import>;*`.
+-   the actor class declaration `<dec>` has function type `(U1, ...​,Un) -> async T` under the static environment induced by the imports in `<import>;*`.
```
Comment (ggreif) [inline doc/md/language-manual.md:1058]:
```suggestion
-   the actor class declaration `<dec>` has function type `(U1, ...​, Un) -> async T` under the static environment induced by the imports in `<import>;*`.
```

File: doc/md/language-manual.md [hunk 4]
```diff
@@ -1067,7 +1067,76 @@ module {
...
+
+where
+
+```
+  type CanisterSettings = {
+     settings : ? {
+        controllers : ? [Principal];
+        compute_allocation : ?Nat;
+        memory_allocation : ?Nat;
+        freezing_threshod : ?Nat;
```
Comment (ggreif) [inline doc/md/language-manual.md:1105]:
```suggestion
        freezing_threshold : ?Nat;
```
please grep for this

File: doc/md/language-manual.md (hunk 1 above)
Comment (ggreif) [reply doc/md/language-manual.md:1070]: I suggest keeping them everywhere: `s/.../…/g`

File: doc/md/language-manual.md [hunk 5]
```diff
@@ -1067,7 +1067,76 @@ module {
...
+     }
+  }
+```
+
+Calling `(system Lib.<id>)(<exp>)(<exp1>, ...​, <expn>)` uses the additional argument `<exp>`, a variant value, to control the installation of the canister further. Arguments `(<exp1>,..., <expn>)` are the user-declared constructor arguments.
```
Comment (ggreif) [inline doc/md/language-manual.md:1110]:
```suggestion
Calling `(system Lib.<id>)(<exp>)(<exp1>, ...​, <expn>)` uses the additional argument `<exp>`, a variant value, to control the installation of the canister further. Arguments `(<exp1>, ..., <expn>)` are the user-declared constructor arguments.
```

**Review by ggreif — APPROVED** [review]
Nice! Nits only.

File: doc/md/language-manual.md (hunk 1 above)
Comment (crusso) [reply doc/md/language-manual.md:1070]: Yeah, I opened an issue about this here: #3418. Not sure whether to use unicode or not but it should at lease be consistent throughout. I wonder if the adoc conversion went wrong...

**Pull request merged**